src/lib/custom-color.js

```diff
@@ -1,39 +1,27 @@
-import color from "color";
-
 const renderCustomColor = function(config) {
 	// Custom styling
-	if (config.customColor) {
-		let lighterColor = color(config.customColor).lighten(0.2).hex();
+	if (config.theme.primaryColor || config.customColor) {
```

Comment:
`Uncaught TypeError: Cannot read property 'primaryColor' of undefined` occurs if `theme` is not set in the config.